src/test/java/be/seeseemelk/mockbukkit/MockChunkDataTest.java

```diff
@@ -0,0 +1,53 @@
+package be.seeseemelk.mockbukkit;
+
+import org.bukkit.Material;
+import org.bukkit.generator.ChunkGenerator;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class MockChunkDataTest
+{
+	private ServerMock server;
+
+	@BeforeEach
+	public void setUp()
+	{
+		server = MockBukkit.mock();
+	}
+
+	@AfterEach
+	public void tearDown()
+	{
+		MockBukkit.unmock();
+	}
+
+	@Test
+	void test_set_and_get()
+	{
+		WorldMock dummy = server.addSimpleWorld("dummy");
+		ChunkGenerator.ChunkData data = server.createChunkData(dummy);
+
+		data.setBlock(0, 0, 0, Material.STONE);
+		Assertions.assertEquals(Material.STONE, data.getType(0, 0, 0));
+	}
+
+	@Test
+	void test_set_out_of_bounds()
+	{
+		WorldMock dummy = server.addSimpleWorld("dummy");
+		ChunkGenerator.ChunkData data = server.createChunkData(dummy);
+
+		data.setBlock(33, 1000, 33, Material.STONE);
+	}
```

Comment:
There are no assertions in this test, at least an `assertDoesNotThrow` would be minimum.